tests/Struct/FindologicProductTest.php

```diff
@@ -2232,4 +2232,34 @@ public function testCategoryAndCatUrlExportBasedOnIntegrationType(
             $this->assertSame($expectedCategories, $attributes[0]->getValues());
         }
     }
+
+    public function testAttributesAreHtmlEntityEncoded(): void
+    {
+        $expectedAttributeValue = '>80';
+        $productEntity = $this->createTestProduct([
+            'customFields' => [
+                'findologic_yeet' => '&gt;80',
```

Comment:
```suggestion
                'length' => '&gt;80',
```

Reply:
I liked `findologic_yeet` :cry:

Reply:
Just for readability, otherwise I don't care ;)